code/datums/mood_events/generic_positive_events.dm

```diff
@@ -36,3 +36,15 @@
 /datum/mood_event/oblivious
 	description = "<span class='nicegreen'>What a lovely day.</span>\n"
 	mood_change = 3
+
+/datum/mood_event/focused
+	description = "<span class='nicegreen'>I have a goal, and I will reach it, whatever it takes!</span>\n" //Used for syndies, nukeops etc so they can focus on their goals
+	mood_change = 12
```

Comment:
i wanted to suggest this but what if people antag test with it? can you visibly check for how sad somebody else is?

Reply:
no, and if I do add the option I will hide this moodlet in the calculations by using a bool like hidden_moodlet.

Reply:
Being able to tell what moodlets people have would actually be a neat trait.

Reply:
free trait for psychologist